instrumentation/github.com/gorilla/mux/otelmux/internal/semconv/gen.go

```diff
@@ -0,0 +1,14 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package semconv // import "go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux/internal/semconv"
+
+// Generate semconv package:
```

Comment:
You will also need to generate the test subfolder.
And to add a `go.mod` and `go.sum` in that test subfolder.